src-docs/src/theme_dark.scss

```diff
@@ -10,9 +10,8 @@
  */
 
 // sass-lint:disable no-url-domains, no-url-protocols
```

Comment:
Would love to get rid of old `sass-lint` disables

Reply:
`sass-lint` have been removed from the codebase